src/html_converter.py

```diff
@@ -0,0 +1,31 @@
+import os
+import sys
+import click
+import html2text
+
+@click.command()
+@click.option('--input_file', '-i', default=None)
+@click.option('--output_dir', '-o', default=None)
+def main(input_file, output_dir):
+    if input_file is None or output_dir is None:
+        print('invalid arguments')
+        sys.exit(1)
+    else:
+        try:
+            with open(input_file, 'r') as f:
+                html = f.read()
+        except:
+            print(f'{input_file} does not exist')
+            sys.exit(1)
+        text = html2text.html2text(html)
+        output = os.path.join(output_dir, os.path.basename(input_file))
```

Comment:
拡張子 `.md` に変更せず `.html` のまま保存しちゃっているのでこうかな？

```suggestion
        output_file = '{}.md'.format(os.path.basename(input_file).split('.')[0])
        output = os.path.join(output_dir, output_file)
```

Reply:
すみません！こちらのコメント見落としていました。。
修正しました:bow: